### ISSUE TITLE
Map Filtering

### ISSUE THREAD

**Issue by miklb** [hurricane-response/florence_website#28]
while introducing the Mapbox GL map, we have opportunity to detail how best to filter and if any other layer data would be useful.

/cc @sbma44

**Comment by miklb** [comment]
For instance, this has evacuation routes layered. Could we show layers based on pre-landfall/post?

**Comment by sbma44** [comment]
Absolutely. Here's an example showing how to toggle layers: https://www.mapbox.com/mapbox-gl-js/example/toggle-layers/

It's also possible to dynamically modify filters that determine which elements within a layer are rendered. This can be used in a lot of interesting ways to create dynamic displays.

(Note that it's probably *not* a good fit for a dataset like the shelters that can change rapidly--adding new shelters does require re-uploading the tileset, which can be automated but involves some async processing)

**Comment by miklb** [comment]
> which can be automated but involves some async processing
is that in reference to just the layers or in general? If a new shelter was added right now, would it display?
